readme.md

```diff
@@ -0,0 +1,37 @@
+# c-multihash
+
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-multiformats-blue.svg?style=flat-square)](http://github.com/multiformats/multiformats)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+
+> C implementation of Multihash parsing and encoding (but not hashing)
+
+This is an implementation of [multihash](https://github.com/multiformats/multihash).
+
+## Install
+
+```
+> TODO
+```
+
+## Usage
+
+```
+> TODO
+```
+
+## Maintainers
+
+Captain: [@Kubuxu](https://github.com/Kubuxu).
+
+## Contribute
+
+Contributions welcome. Please check out [the issues](https://github.com/multiformats/c-multihash/issues).
+
+Check out our [contributing document](https://github.com/multiformats/multiformats/blob/master/contributing.md) for more information on how we work, and about contributing in general. Please be aware that all interactions related to multiformats are subject to the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+Small note: If editing the Readme, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
+
+## License
+
+[MIT](LICENSE) © Kubuxu
```

Comment:
- i think we should do `Protocol Labs, Inc` for now, and later move to `IPFS Authors` once i figure out how such an can receive copyright assignments